modules/API/pages/built-in-endpoints.adoc

```diff
@@ -1297,9 +1376,9 @@ This endpoint upserts vertices and/or edges into a graph. To upsert means that i
 
 ==== Atomic upsert transaction
 
-By default, the `+POST /graph/{graph_name}+` endpoint is not atomic. If something goes wrong during the process of the request,  the request data can be partially consumed by the database.
+By default, the `+POST /graph/{graph_name}+` endpoint is not atomic. If something goes wrong during the server processing stage of the request, the request data can be partially consumed by the database.
```

Comment:
Let's not narrow down the stage when something could go wrong. Something could go wrong during any stage of the request, making the request, server processing the request, anything. I would just say: 

"If anything goes wrong during the lifecycle of a request, the request data can be partially consumed by the database."